### PR TITLE
Update migrate-from-csv-using-tidb-lightning.md

### PR DESCRIPTION
**Pull request by lance6716**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ title: CSV 支持
 aliases: ['/docs-cn/dev/tidb-lightning/migrate-from-csv-using-tidb-lightning/','/docs-cn/dev/reference/tools/tidb-lightning/csv/']
 ---
 
-# CSV 支持
+本文介绍如何使用 TiDB Lightning 从 CSV 文件迁移数据到 TiDB。关于如何从 MySQL 生成 CSV 文件，可以参考[使用 Dumpling 导出到 CSV 文件](/dumpling-overview.md#导出到-csv-文件)。
+
+# CSV 支持与限制
 
 TiDB Lightning 支持读取 CSV（逗号分隔值）的数据源，以及其他定界符格式如 TSV（制表符分隔值）。
```
Comment (TomShawn) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:11]:
```suggestion
# CSV 支持与限制

TiDB Lightning 支持读取 CSV（逗号分隔值）的数据源，以及其他定界符格式如 TSV（制表符分隔值）。

本文介绍如何使用 TiDB Lightning 从 CSV 文件迁移数据到 TiDB。关于如何从 MySQL 生成 CSV 文件，可以参考[使用 Dumpling 导出到 CSV 文件](/dumpling-overview.md#导出到-csv-文件)。

```

Comment (lance6716) [reply tidb-lightning/migrate-from-csv-using-tidb-lightning.md:11]: 是想跟 SQL 文件导入一样，一上来有一句介绍

Comment (hfxsd) [reply tidb-lightning/migrate-from-csv-using-tidb-lightning.md:11]: OK。我把Tom comment中的两段的位置进行了对调。这样那句介绍的内容就放在第一段了。

**Review by hfxsd — APPROVED** [review]

**Review by sunzhaoyang — APPROVED** [review]

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 2]
```diff
@@ -3,10 +3,11 @@ title: CSV 支持
...
 ---
 
-# CSV 支持
+# CSV 支持与限制
 
-TiDB Lightning 支持读取 CSV（逗号分隔值）的数据源，以及其他定界符格式如 TSV（制表符分隔值）。
+本文介绍如何使用 TiDB Lightning 从 CSV 文件迁移数据到 TiDB。关于如何从 MySQL 生成 CSV 文件，可以参考[使用 Dumpling 导出到 CSV 文件](/dumpling-overview.md#导出到-csv-文件)。
 
+TiDB Lightning 支持读取 CSV（逗号分隔值）的数据源，以及其他定界符格式如 TSV（制表符分隔值）。
 ## 文件名
```
Comment (hfxsd) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:11]:
```suggestion

## 文件名
```

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**